doc/user/layouts/shortcodes/rolling-out-inline.html

```diff
@@ -0,0 +1,12 @@
+{{$version := .Get 0}}
+{{$releasePages := where $.Site.RegularPages "Section" "releases"}}
+{{$releasePage := (index (where $releasePages ".File.ContentBaseName" $version) 0)}}
+{{$isReleased := $releasePage.Params.released}}
+{{$releaseDate := $releasePage.Params.date}}
+
+{{if not $isReleased}}
+<strong>Rolling out: </strong>This feature is currently rolling out
+in <a href="{{$releasePage.RelPermalink}}"><strong>{{$version}}</strong></a>.
+It may not be available in your region yet.
+Rollout is scheduled to complete by <strong>{{dateFormat "January 2, 2006" $releaseDate}}</strong>.
+{{end}}
```

Comment:
```suggestion
<strong>Unreleased: </strong>This feature will be released
in <a href="{{$releasePage.RelPermalink}}"><strong>{{$version}}</strong></a>.
It may not be available in your region yet.
The release is scheduled to complete by <strong>{{dateFormat "January 2, 2006" $releaseDate}}</strong>.
{{end}}
```